docs/team/cnivedit.md

```diff
@@ -0,0 +1,58 @@
+# Chittazhi Nivedit Nandakumar - Project Portfolio Page
+
+## Overview
+Pharmacy Inventory & Logistics Ledger (PILL) is a desktop application that allows
+pharmacists to keep track of and manage medicinal inventory.
+
+PILL is fully written in Java, and users can interact with the application using
+Command Line Interface (CLI).
+
+### Summary of Contributions
+
+- **New Feature**: Added ListCommand class
+    - What it does: The class is responsible for handling the execution of the list command ([#7](https://github.com/AY2425S1-CS2113-W14-4/tp/pull/7)).
+    - Justification: Abstraction of the list command, inheriting from the base command class.
+    - Highlights: Receives ItemMap instance and lists items.
```

Comment:
Looks good, but you could consider making more difficult to implement features take up more space/be at the top, such that the most impressive features are easier to notice

Reply:
Updated list order in cf91040